packages/react/src/components/GaugeCard/GaugeCard.jsx

```diff
@@ -151,7 +151,7 @@ const GaugeCard = ({
                       { [`${iotPrefix}--gauge-value-lg`]: valueLength <= 2 }
                     )}
                     x={gaugeSize / 2}
-                    y="33"
+                    y={gaugeSize / 2 + 8}
```

Comment:
nit: I would put the stroke value in a const

Reply:
this was actually already done. You were reviewing it while I pushed the updates from David's comments. :-)